doc/tutorial/using_multi_gpu.txt

```diff
@@ -0,0 +1,149 @@
+
+.. _tut_using_multi_gpu:
+
+===================
+Using multiple GPUs
+===================
+
+Theano has a feature to allow the use of multiple GPUs at the same
+time in one function.  The multiple gpu feature requires the use of
+the :ref:`gpuarray` backend, so make sure that works correctly.
+
+In order to keep a reasonably high level of abstraction you do not
+refer to device names directly for multiple-gpu use.  You instead
+referer to what we call context names.  These are then mapped to a
+device using the theano configuration.  This allows portability of
+models between machines.
+
+.. warning::
+
+   The code is rather new and is still considered experimental at this
+   point.  It has been tested and seems to perform correctly in all
+   cases observed, but make sure to double-check your results before
+   publishing a paper or anything of the sort.
+
+.. warning::
+
+   Due to some implementation issues, models using multiple GPUs will,
+   in most cases not exhibit any speedup over running on a single GPU.
+   These issues are being worked on and should be solved within 2-3
+   weeks.  We do not expect any interface change, so you can still
```

Comment:
no. It wasn't updated, but this is fixed.

On Thu, Nov 26, 2015 at 9:08 AM, Dzmitry Bahdanau <notifications@github.com>
wrote:

> In doc/tutorial/using_multi_gpu.txt
> <https://github.com/Theano/Theano/pull/3559#discussion_r45981288>:
>
> > +device using the theano configuration.  This allows portability of
> > +models between machines.
> > +
> > +.. warning::
> > +
> > +   The code is rather new and is still considered experimental at this
> > +   point.  It has been tested and seems to perform correctly in all
> > +   cases observed, but make sure to double-check your results before
> > +   publishing a paper or anything of the sort.
> > +
> > +.. warning::
> > +
> > +   Due to some implementation issues, models using multiple GPUs will,
> > +   in most cases not exhibit any speedup over running on a single GPU.
> > +   These issues are being worked on and should be solved within 2-3
> > +   weeks.  We do not expect any interface change, so you can still
>
> Is it still relevant?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3559/files#r45981288>.
>